requirements.txt

```
@@ -5,4 +5,5 @@ python-dotenv   == 1.0.1
 requests        == 2.32.3
 retry           == 0.9.2
 statistics      == 1.0.3.5
-xxhash          == 3.5.0
\ No newline at end of file
+xxhash          == 3.5.0
+flake8
```

I think latest is `7.1.0`

```suggestion
flake8 == 7.1.0
```